 libbinder: fix unresolved symbol WriteString16
until bionic doesn't properly load shim libs, use this workaround

diff --git a/libs/binder/Parcel.cpp b/libs/binder/Parcel.cpp
index 87e5b40..9634db7 100644
--- a/libs/binder/Parcel.cpp
+++ b/libs/binder/Parcel.cpp
@@ -2753,3 +2753,5 @@ void Parcel::Blob::clear() {
 }
 
 }; // namespace android
+
+__strong_alias(_ZN7android6Parcel13writeString16EPKtj,_ZN7android6Parcel13writeString16EPKDsj);
